clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -156,6 +156,13 @@ get_current_no_trace_for_instrs_value()
     return 0;
 }
 
+// This function returns true if there exists an instruction count threshold to enable
+// tracing. A value of 0 from get_current_no_trace_for_instrs_value() means the
```

Comment:
I feel treating "0" in such different ways is not ideal and adds to the complexity of the implementation. See other comments.